client/components/date-range/test/index.js

```diff
@@ -775,89 +676,41 @@ describe( 'DateRange', () => {
 				/>
 			);
 
-			const inputs = wrapper.find( DateRangeInputs );
-			const resetBtn = wrapper.find( '.date-range__info-btn' );
+			await userEvent.click( screen.getByLabelText( 'Select date range' ) );
 
-			inputs.props().onInputBlur( '03/20/2018', 'Start' );
-			inputs.props().onInputBlur( '09/20/2018', 'End' );
+			const fromInputEl = screen.getByLabelText( 'From' );
+			const toInputEl = screen.getByLabelText( 'To' );
 
-			expect( dateToLocaleString( wrapper.state().startDate ) ).toEqual(
-				dateToLocaleString( '03/20/2018' )
-			);
-			expect( dateToLocaleString( wrapper.state().endDate ) ).toEqual(
-				dateToLocaleString( '09/20/2018' )
-			);
-
-			// Now click to "clear" or "reset" the selection
-			resetBtn.simulate( 'click' );
-
-			expect( dateToLocaleString( wrapper.state().startDate ) ).toEqual(
-				dateToLocaleString( '04/28/2018' )
-			);
-			expect( dateToLocaleString( wrapper.state().endDate ) ).toEqual(
-				dateToLocaleString( '05/28/2018' )
-			);
-		} );
-	} );
-
-	describe( 'Render props', () => {
-		test( 'should allow for render prop to overide trigger render', () => {
-			const spyComponent = jest.fn();
-
-			shallow(
-				<DateRange translate={ translate } moment={ moment } renderTrigger={ spyComponent } />
-			);
+			await userEvent.click( fromInputEl );
+			await userEvent.clear( fromInputEl );
+			await userEvent.keyboard( '04/13/2018' );
+			await userEvent.click( toInputEl );
+			await userEvent.clear( toInputEl );
+			await userEvent.keyboard( '05/13/2018' );
+			// force a blur out of `toEl`
+			await userEvent.click( fromInputEl );
```

Comment:
I think `userEvent.tab()` will be more idiomatic here.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/63879/commits/03c92dead8665fdecde8e81b67f5a54bad3f6cfa.